app/controllers/api/v1/interactive_pages_controller.rb

```diff
@@ -37,7 +37,7 @@ def copy_page
 
     return error("Can't find activity for page") unless activity
 
-    position = params[:dest_index];
+    position = params[:dest_index]
```

Comment:
Did you mean to delete this semicolon?

Reply:
Yeah, I was fixing some  Ruby ⟷ Javascript whiplash I was suffering with.  Ruby doesn't typically use semicolons like this.  :D

Reply:
Ah, right! This is a Ruby file. Duh :)